nropf.js

```diff
@@ -15,8 +15,8 @@ exports.use = function( filePath ){
     if ( trimmedLine === "") return;
 
     var keyAndValue = trimmedLine.split("=");
-    var key = keyAndValue[0].trim();
-    var value = keyAndValue[1].trim();
+    var key = keyAndValue.shift().trim();
+    var value = keyAndValue.join("=").trim();
```

Comment:
The way you did it works quite fine.
Can you propose a version of it that would not mutate the keyAndValue var ?


I'd be happy to merge it then